manifest.json

```diff
@@ -26,6 +26,8 @@
   ],
   "dependencies": {
     "vtex.styleguide": "6.x",
-    "vtex.product-summary": "1.x"
+    "vtex.product-summary": "1.x",
+    "vtex.store-graphql": "2.x",
```

Comment:
remove it

Reply:
I'm sorry, Bruno... I'm afraid I can't do that...